### PR TITLE
update careers page

### PR DESCRIPTION
**Pull request by argshook**
This PR updates the careers page

- refactor repetitive HTML to instead loop over data structure
- update current open positions

### CODE REVIEW

File: src/pages/careers.tsx [hunk 1]
```diff
@@ -92,108 +131,33 @@ const CareersPage = () => {
...
-                Details&nbsp;
-                <img
-                  alt="Right arrow"
-                  height={20}
-                  src="/img/pages/careers/arrowRight.svg"
-                  width={20}
-                />
-              </span>
-            </a>
+            {currentOpenings.map(({ href, title, location }) => (
```
Comment (bsmth) [inline src/pages/careers.tsx:143]: nice

**Review by bsmth — CHANGES_REQUESTED** [review]
Maybe we should add redirects to the missing pages at

https://github.com/questdb/questdb.io/blob/master/netlify.toml

We can redirect to `/careers` maybe?

**Comment by argshook** [comment]
> Maybe we should add redirects to the missing pages at
> 
> https://github.com/questdb/questdb.io/blob/master/netlify.toml
> 
> We can redirect to `/careers` maybe?

good idea, updated!

**Review by bsmth — APPROVED** [review]
Thanks 👍🏻

**Pull request merged**